src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +46,12 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Movie> getAllMovie =
+                    session.createQuery("from movie", Movie.class);
```

Comment:
The HQL query string should use the class name `Movie` instead of `movie`. The correct query should be `from Movie` to match the entity class name.